packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -116,14 +133,24 @@ export default {
       )
     },
     limitedPersonalStorage() {
+      if (!this.useLegacyQuota) {
+        return this.quota.total !== 0
```

Comment:
Evaluates to `true` if the `total` field is absent. Afaik this should evaluate to `false` then.

Reply:
Total 0 means unlimited in the ocis world

Reply:
But if `total` is not specified I'd interpret it as the same as `0`. Needs to be clarified then I guess. 😅

Reply:
Yes you are right if 0 or not specified, means unlimited, was just a note for me